src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected(tab: Tab): void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTabId = tabs
+    .find(tab => tab.id === selectedTabId)?.id || tabs[0].id;
```

Comment:
```suggestion
  const currentTab = tabs
    .find(tab => tab.id === selectedTabId)? || tabs[0];
```
lets keep the whole current tab and use its `content` and `id` where it is needed 😉